source/Grid/Grid.js

```diff
@@ -427,6 +428,12 @@ export default class Grid extends Component {
           let columnDatum = this._columnMetadata[columnIndex]
           let renderedCell = renderCell({ columnIndex, rowIndex })
           let key = `${rowIndex}-${columnIndex}`
+
+          // any other falsey value will be rendered
+          // as a text node by React
+          if (renderedCell == null || renderedCell === false)
```

Comment:
Its a bit of an optimization and helpful in the ScrollSync case, you can see in the updated Example the renderBodyCell method doesn't render cells that are "locked" and displayed in the scynced grid overlayed on the left. Not that having the duplication would work, but this is a bit cleaner i think.